polaris-react/src/tokens/token-groups/typography.json

```diff
@@ -1,31 +0,0 @@
-{
-  "font-family-sans": "-apple-system, BlinkMacSystemFont, 'San Francisco', 'Segoe UI', Roboto, 'Helvetica Neue', sans-serif",
-  "font-family-mono": "ui-monospace, SFMono-Regular, 'SF Mono', Consolas, 'Liberation Mono', Menlo, monospace",
```

Comment:
Is this a breaking change as the built files no longer include the tokens? https://unpkg.com/browse/@shopify/polaris@9.9.0/build/esm/tokens/tokens.js

I don't think this was ever documented to consumers but it might be a concern. Is there any code that we should remove that adds them to the build?

Reply:
Shouldn't be. We intentionally didn't add `tokens` to the public API: https://unpkg.com/browse/@shopify/polaris@9.9.0/build/esm/index.js

Reply:
Awesome. Thanks @aaronccasanova